### ISSUE TITLE
<au-code language="markup"> custom element should accept "hard line breaks"

### ISSUE THREAD

**Issue by adriatic** [aurelia-ui-toolkits/aurelia-kendoui-plugin#146]
At the moment `markup` like this:
```html
<au-code language="markup">
  <au-kendo-chart title.bind="{text: 'Internet Users in United States'}"
                  data-source.bind="{data: internetUsers}"
                  legend.bind="{ visible: false }"
                  series-defaults.bind="seriesDefaults"
                  series.bind="series"
                  value-axis.bind="valueAxis"
                  category-axis.bind="categoryAxis"
                  tooltip.bind="tooltip">
  </au-kendo-chart>
</au-code>
```

is rendered like this:

![image](https://cloud.githubusercontent.com/assets/2712405/11796800/5c090c3c-a28c-11e5-811d-0c3edb2e8c5e.png)

**Comment by JeroenVinke** [comment]
Hmm.. I wonder if having a wysiwyg code editor would be a better solution. We could put the sample code  in an editor and get the html from it, and put that in a div.

**Comment by adriatic** [comment]
I would first consult Aike, who I think wrote the component we are using now.

**Comment by JeroenVinke** [comment]
@adriatic can you find out how Aurelia is displaying code on their docs? perhaps we can use the same code/method

**Comment by adriatic** [comment]
Will do (for the second time - I did that a month ago and Rob responded that Aurelia's code for that is too shaky to share with anyone).

**Comment by JeroenVinke** [comment]
fixed by 386676efaae769e2d7ce2707d747028f9d2e845a
